cgk_calcium_tools/filehandler.py

```diff
@@ -2,242 +2,318 @@
-                self.focus_files[rec]=[str(Path(self.outputsfolders[i])/Path(ef))  
-                                            for ef in efocus_filenames]
-                self.p_recording_labels.extend([self.recording_labels[i]+f'_{ef}'
-                                                    for ef in efocus])
+                efocus_filenames = [
+                    pr.stem + "_" + str(ef) + pr.suffix for ef in efocus
+                ]
+                self.focus_files[rec] = [
+                    str(Path(self.outputsfolders[i]) / Path(ef))
+                    for ef in efocus_filenames
+                ]
+                self.p_recording_labels.extend(
+                    [self.recording_labels[i] + f"_{ef}" for ef in efocus]
+                )
                 self.p_rec_paths.extend(self.focus_files[rec])
-            self.p_outputsfolders.extend([self.outputsfolders[i]]*len(efocus))
-        
+            self.p_outputsfolders.extend(
+                [self.outputsfolders[i]] * len(efocus))
+
         if files_list_log is not None:
-            file_handler_status={
-                'input_parameters':input_parameters,  
-                'computed_parameters':{
-                    'rec_subfolders':self.rec_subfolders,
-                    'outputsfolders':self.outputsfolders,
-                    'recording_labels':self.recording_labels,
-                    'rec_paths':self.rec_paths,
-                    'p_rec_paths':self.p_rec_paths,
-                    'p_outputsfolders':self.p_outputsfolders,
-                    'p_recording_labels':self.p_recording_labels,
-                    'focus_files':self.focus_files, 
-                    'efocus':self.efocus,
-                }}
-            with open(files_list_log, 'w') as file:
+            file_handler_status = {
+                "input_parameters": input_parameters,
+                "computed_parameters": {
+                    "rec_subfolders": self.rec_subfolders,
+                    "outputsfolders": self.outputsfolders,
+                    "recording_labels": self.recording_labels,
+                    "rec_paths": self.rec_paths,
+                    "p_rec_paths": self.p_rec_paths,
+                    "p_outputsfolders": self.p_outputsfolders,
+                    "p_recording_labels": self.p_recording_labels,
+                    "focus_files": self.focus_files,
+                    "efocus": self.efocus,
+                },
+            }
+            with open(files_list_log, "w") as file:
                 json.dump(file_handler_status, file)
 
-    def de_interleave(self, overwrite:bool=False)->None:
-        print('de_interleaving movies, please wait...')
-        for (main_file,planes_fs), focus in zip(self.focus_files.items(), self.efocus):
-            if len(focus)>1: #has multiplane
+    def de_interleave(self, overwrite: bool = False) -> None:
+        """
+        This funtion use the isx.de_interleave funtion, wich de-interleaves multiplane movies
```

Comment:
can you change it to: 

This function applies the isx.de_interleave function, which de-interleaves multiplane movies